README.md

```diff
@@ -80,6 +65,22 @@ Finally, enable all of the rules that you would like to use.  Use [our preset](#
   }
 ```
 
+You can also specify settings that will be shared across all the plugin rules.
```

Comment:
We should probably put this higher up, and change "can" to "should" - the react version in particular is pretty important.